### PR TITLE
Improvement of requirement/build information

### PR DESCRIPTION
**Pull request by gertingold**
This PR implements changes as discussed in #335.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -51,20 +49,12 @@ Requirements
 * pandas
 * seaborn
 
+For a complete list of requirements with information on the versions presently used
+to build the SciPy lecture notes, see ``requirements.txt``.
```
Comment (GaelVaroquaux) [inline CONTRIBUTING.rst]: Maybe a `:download:` link, so that whoever is looking at that on the web can download the requirements.txt file?

Comment (gertingold) [reply CONTRIBUTING.rst]: Good point. An alternative would be to use a `literalinclude` of the requirements file. What do you think?

**Comment by gertingold** [comment]
Now, `requirements.txt` is inserted by a `literalinclude`, thus keeping the requirements list always up-to-date.

**Comment by GaelVaroquaux** [comment]
Looks great! Merging. Thank you

**Pull request merged**